sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/CreateUserTest.java

```diff
@@ -21,257 +29,513 @@
-
-	}
-
-	@Test(priority = 6)
-	@CitrusTest
-	public void getUserTest() {
-		http().client(restTestClient).send()
-				.get("/api/user/v1/read/" + userId + "?Fields=completeness,missingFields,topic")
-				.accept(ConstantKeys.CONTENT_TYPE_APPLICATION_JSON)
-				.header(ConstantKeys.AUTHORIZATION, ConstantKeys.BEARER + initGlobalValues.getApiKey())
-				.contentType(ConstantKeys.CONTENT_TYPE_APPLICATION_JSON)
-				.header(ConstantKeys.X_AUTHENTICATED_USER_TOKEN, user_auth_token);
-		http().client(restTestClient).receive().response(HttpStatus.OK)
-				.validationCallback(new JsonMappingValidationCallback<Response>(Response.class, objectMapper) {
-					@Override
-					public void validate(Response response, Map<String, Object> headers, TestContext context) {
-						Assert.assertNotNull(response.getId());
-						Assert.assertEquals(response.getResponseCode(), ResponseCode.OK);
-					}
-				});
-	}
-
-	private String createUserMap() {
-		Map<String, Object> requestMap = new HashMap<>();
-		requestMap.put("request", createUserInnerMap());
-		try {
-			return objectMapper.writeValueAsString(requestMap);
-		} catch (JsonProcessingException e) {
-			e.printStackTrace();
-		}
-		return null;
-	}
-
-	private String createUserWithDuplicateUserName() {
-		Map<String, Object> requestMap = new HashMap<>();
-		Map<String, Object> innerMap = createUserInnerMap();
-		innerMap.put("email", ConstantKeys.USER_NAME_PREFIX + UUID.randomUUID().toString() + "@gmail.com");
-		requestMap.put("request", innerMap);
-		try {
-			return objectMapper.writeValueAsString(requestMap);
-		} catch (JsonProcessingException e) {
-			e.printStackTrace();
-		}
-		return null;
-	}
-
-	private String createUserWithDuplicateEmail() {
-		Map<String, Object> requestMap = new HashMap<>();
-		Map<String, Object> innerMap = createUserInnerMap();
-		innerMap.put("userName", ConstantKeys.USER_NAME_PREFIX + UUID.randomUUID().toString());
-		requestMap.put("request", innerMap);
-		try {
-			return objectMapper.writeValueAsString(requestMap);
-		} catch (JsonProcessingException e) {
-			e.printStackTrace();
-		}
-		return null;
-	}
-
-	private static Map<String, Object> createUserInnerMap() {
-		Map<String, Object> innerMap = new HashMap<>();
-		innerMap.put("firstName", "ft_first_Name_pw12401");
-		innerMap.put("lastName", "ft_lastName");
-		innerMap.put("password", "password");
-		USER_NAME = ConstantKeys.USER_NAME_PREFIX + EndpointConfig.val;
-		String email = ConstantKeys.USER_NAME_PREFIX + EndpointConfig.val + "@gmail.com";
-		innerMap.put("userName", USER_NAME);
-		innerMap.put("email", email);
-		return innerMap;
-	}
+  private static String userId = null;
+  private static String user_auth_token = null;
+  private static String admin_token = null;
+  public static Map<String, List<String>> deletedRecordsMap = new HashMap<String, List<String>>();
+  private static final String CREATE_USER_URI = "/api/user/v1/create";
+  private static final String UPDATE_USER_URI = "/api/user/v1/update";
```

Comment:
for update User test write a separate class as UpdateUserTest , that will have all the methods for user update test scenario.